src/transformers/utils/__init__.py

```diff
@@ -299,8 +298,8 @@ def check_min_version(min_version):
         )
 
 
-@lru_cache()
-def get_available_devices() -> FrozenSet[str]:
+@lru_cache
+def get_available_devices() -> frozenset[str]:
```

Comment:
I notice this file still has the `FrozenSet` type import! Any idea why?

Reply:
ruff was used to remove unused import in most files. But __init__.py files are a bit special and their imports are used outside.

Reply:
Makes sense!